app/controllers/branches_controller.rb

```diff
@@ -40,6 +40,6 @@ def destroy
   end
 
   private def branch_params
-    params.require(:branch).permit(:name, :default, :opening_time, :closing_time)
+    params.require(:branch).permit(:name, :default, :opening_time, :closing_time, address_attributes: [:address_line_1, :address_line_2, :city, :state, :pincode, :_destroy])
```

Comment:
We are not allowing destroying branch address.
Fix this

Reply:
Done